src/main/java/duke/exception/InvalidTimeAndDateException.java

```diff
@@ -3,8 +3,17 @@
 public class InvalidTimeAndDateException extends Exception {
     private static final String MESSAGE = "☹ OOPS!!! You are breaking the space-time continuum!";
```

Comment:
I like the consideration of making this a "private static final" constant. Very good, something I shall adapt as well! Well done!